engines/hugo/dialogs.cpp

```diff
@@ -56,15 +56,15 @@ void TopMenu::init() {
 	int x = kMenuX;
 	int y = kMenuY;
 
-	_whatButton = new GUI::PicButtonWidget(this, x, y, kButtonWidth, kButtonHeight, "What is it?", kCmdWhat);
-	_musicButton = new GUI::PicButtonWidget(this, x, y, kButtonWidth, kButtonHeight, "Music", kCmdMusic);
-	_soundFXButton = new GUI::PicButtonWidget(this, x, y, kButtonWidth, kButtonHeight, "Sound FX", kCmdSoundFX);
-	_saveButton = new GUI::PicButtonWidget(this, x, y, kButtonWidth, kButtonHeight, "Save game", kCmdSave);
-	_loadButton = new GUI::PicButtonWidget(this, x, y, kButtonWidth, kButtonHeight, "Load game", kCmdLoad);
-	_recallButton = new GUI::PicButtonWidget(this, x, y, kButtonWidth, kButtonHeight, "Recall last command", kCmdRecall);
-	_turboButton = new GUI::PicButtonWidget(this, x, y, kButtonWidth, kButtonHeight, "Turbo", kCmdTurbo);
-	_lookButton = new GUI::PicButtonWidget(this, x, y, kButtonWidth, kButtonHeight, "Description of the scene", kCmdLook);
-	_inventButton = new GUI::PicButtonWidget(this, x, y, kButtonWidth, kButtonHeight, "Inventory", kCmdInvent);
+	_whatButton =		new GUI::PicButtonWidget(this,	x, y, kButtonWidth, kButtonHeight,	Common::U32String("What is it?"),				kCmdWhat);
+	_musicButton =		new GUI::PicButtonWidget(this,	x, y, kButtonWidth, kButtonHeight,	Common::U32String("Music"),						kCmdMusic);
+	_soundFXButton =	new GUI::PicButtonWidget(this,	x, y, kButtonWidth, kButtonHeight,	Common::U32String("Sound FX"),					kCmdSoundFX);
+	_saveButton =		new GUI::PicButtonWidget(this,	x, y, kButtonWidth, kButtonHeight,	Common::U32String("Save game"),					kCmdSave);
+	_loadButton =		new GUI::PicButtonWidget(this,	x, y, kButtonWidth, kButtonHeight,	Common::U32String("Load game"),					kCmdLoad);
+	_recallButton =		new GUI::PicButtonWidget(this,	x, y, kButtonWidth, kButtonHeight,	Common::U32String("Recall last command"),		kCmdRecall);
+	_turboButton =		new GUI::PicButtonWidget(this,	x, y, kButtonWidth, kButtonHeight,	Common::U32String("Turbo"),						kCmdTurbo);
+	_lookButton =		new GUI::PicButtonWidget(this,	x, y, kButtonWidth, kButtonHeight,	Common::U32String("Description of the scene"),	kCmdLook);
+	_inventButton =		new GUI::PicButtonWidget(this,	x, y, kButtonWidth, kButtonHeight,	Common::U32String("Inventory"),					kCmdInvent);
```

Comment:
Are those lines supposed to be aligned? They are not aligned for me as I probably have a different setting than you for the tab size (e.g 2 characters, 4 characters, or 8 characters). Aligning those kind of line works better if you use spaces rather than tabs.